dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/synch/DefaultSynchronizationManager.java

```diff
@@ -231,17 +239,26 @@ public ImportSummaries executeEventPush() throws WebMessageParseException
 
         final Date startTime = new Date();
         final Date lastSuccessTime = getLastEventSynchSuccessFallback();
+        final Date skipChangedBefore = (Date) systemSettingManager
+            .getSystemSetting( SettingKey.SKIP_SYNCHRONIZATION_FOR_DATA_CHANGED_BEFORE );
+        final Date lastUpdatedAfter = lastSuccessTime.after( skipChangedBefore ) ? lastSuccessTime : skipChangedBefore;
+        final int lastUpdatedEventsCount = eventService .getAnonymousEventValuesCountLastUpdatedAfter( lastUpdatedAfter );
 
-        int lastUpdatedEventsCount = eventService.getAnonymousEventValuesCountLastUpdatedAfter( lastSuccessTime );
-
-        log.info( "Events: " + lastUpdatedEventsCount + " since last synch success: " + lastSuccessTime );
+        log.info( "Events last changed before " + skipChangedBefore + " will not be synchronized." );
 
         if ( lastUpdatedEventsCount == 0 )
         {
-            log.info( "Skipping synch, no new or updated data values for events" );
-            return null;
+            SyncUtils.setLastSyncSuccess( systemSettingManager, SettingKey.LAST_SUCCESSFUL_EVENT_DATA_SYNC, startTime );
+            log.info( "Skipping events push, no new or updated events" );
+
+            ImportCount importCount = new ImportCount( 0, 0, 0, 0 );
+            ImportSummary importSummary = new ImportSummary( ImportStatus.SUCCESS, "No new or updated events to push.",
+                importCount );
+            return new ImportSummaries().addImportSummary( importSummary );
         }
 
+        log.info( "Events: " + lastUpdatedEventsCount + " since last synch success: " + lastSuccessTime );
```

Comment:
If I remember correctly, you wanted to use "sync" consistently for logging, and not synch?